.gitignore

```diff
@@ -3,3 +3,4 @@ docker-compose.pgsql.yml
 traefik.toml
 .env
 certs
+.testing
```

Comment:
I think POSIX require newline at the end of a file ;)

Reply:
added in my next commit